src/PKSim.R/Services/PopulationFactory.cs

```diff
@@ -41,11 +40,11 @@ public class PopulationFactory : IPopulationFactory
          _ontogenyVariabilityUpdater = ontogenyVariabilityUpdater;
       }
 
-      public IndividualValuesCache CreatePopulation(PopulationCharacteristics populationCharacteristics)
+      public CreatePopulationResults CreatePopulation(PopulationCharacteristics populationCharacteristics)
       {
          var populationSettings = _populationSettingsMapper.MapToModel(populationCharacteristics).Result;
          validate(populationSettings);
-         var population = _randomPopulationFactory.CreateFor(populationSettings, new CancellationToken()).Result;
+         var population = _randomPopulationFactory.CreateFor(populationSettings, new CancellationToken(), seed: populationCharacteristics.Seed).Result;
```

Comment:
This was a bug here also. The seed was not used....impossible to reproduce the same population